RELEASE-NOTES.md

```diff
@@ -33,6 +33,7 @@
   Removed incorrect copying of connections from `connections.toml` to `config.toml`.
 * Fixes `snow connection generate-jwt` to work with keys with no passphrase.
 * `snow connection add` supports `--no-interactive` flag to skip interactive prompts.
+* The privilege to create a schema or stage is no longer required to run `snow app version create`.
```

Comment:
do we need to specify that this is only if the schema/stage already exist?

Reply:
updated